contracts/peaks/curve/CurveSusdPeak.sol

```diff
@@ -34,6 +34,35 @@ contract CurveSusdPeak is OwnableProxy, Initializable, IPeak {
     IMintr mintr;
     ICore core;
 
+    function migrate(address destination) public onlyOwner {
+        // withdraw from gauge
+        uint sCrv = gauge.balanceOf(address(this));
+        gauge.withdraw(sCrv, false);
+
+        // remove liquidity from sPool
+        sCrv = curveToken.balanceOf(address(this));
+        curve.remove_liquidity(sCrv, ZEROES);
+
+        // swap sUSD for tether
+        uint sUSD = IERC20(underlyingCoins[3]).balanceOf((address(this)));
+        IERC20(underlyingCoins[3]).safeApprove(address(curve), 0);
+        IERC20(underlyingCoins[3]).safeApprove(address(curve), sUSD);
+        curve.exchange(int128(3), int128(2), sUSD, 0);
+
+        // Add liquidity to yPool
+        ICurveDeposit y = ICurveDeposit(0xbBC81d23Ea2c3ec7e56D39296F0cbB648873a5d3);
```

Comment:
Does yPool deposit contract need to be declared in migrate function?

Reply:
It's not trivial to add storage variables while upgrading contracts, so need to work with this.